layers/+themes/theming/README.org

```diff
@@ -28,6 +28,30 @@ of the following form:
            (face2 attributes...)
            ...)
    ...)
+   
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
+#+begin_src lisp
+  ;; ~/.spacemacs
+  ;;...
+  (defun dotspacemacs/user-init ()
+  ;;...
+   (setq-default 
+     ;;... other vars maybe
+     theming-modifications
+      '((monokai
+          ;; Font locking
+          (font-lock-comment-face :slant italic)
+          (web-mode-html-attr-name-face
+           :inherit font-lock-variable-name-face :foreground nil)
+      
+          ;; Modeline
+          (powerline-active1 :box (:color "#999999" :line-width 1 :style released-button)
+                             :background "#5a5a5a"))))
+  )
```

Comment:
Would be nicer to move this paren to the previous line and join it to the other parens.